README.md

```diff
@@ -3,5 +3,49 @@ golang-docker
 
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/golang-docker.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/golang-docker)
 
-This repository contains the sources for the following [docker](https://docker.io) base images:
-- [`gcr.io/google_appengine/golang`](/base)
+This repository contains the source for the `gcr.io/google_appengine/golang` [Docker](https://docker.io) base image.
+It serves as a base for [Go](https://golang.org) apps running on Google App Engine
+[Managed VMs](https://cloud.google.com/appengine/docs/go/managed-vms/).
+
+## Notes
+
+### Vendoring
+
+Your application will be built using Go 1.6 inside the docker instance without
+the experimental vendoring support.  To enable Go 1.6 experimental vendoring
+support, set the environment variable like this:
+
+        FROM gcr.io/google_appengine/golang
+        ENV GO15VENDOREXPERIMENT 1
+        COPY . /go/src/app
+        RUN go-wrapper install -tags appenginevm
```

Comment:
The whole section. Since we're on 1.6 now, GO15VENDOREXPERIMENT isn't worth a mention now.